### PR TITLE
Albums api integration

### PR DESCRIPTION
**Pull request by Marcosnr97**
### Summary
- Added API integration with JSONPlaceholder
- Added getAlbums and getPhotos endpoints

### Trello Card
https://trello.com/c/DPv9X2cl/34-integraci%C3%B3n-con-api-de-albumes-servicios-promises

### CODE REVIEW

File: app/middlewares/errors.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ const DEFAULT_STATUS_CODE = 500;
 
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
-  [errors.DEFAULT_ERROR]: 500
+  [errors.DEFAULT_ERROR]: 500,
+  [errors.EXTERNAL_API_ERROR]: 502
```
Comment (jersonlopez) [inline app/middlewares/errors.js]: I think 503 status is better, what do you thing?

File: app/services/albums.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const options = {
+  uri: `${api_url}`,
+  headers: {
+    'User-Agent': 'Request-Promise'
+  },
+  json: true
+};
+
+exports.getAlbums = id => {
+  if (id) options.uri = `${api_url}/${id}/photos`;
```
Comment (jersonlopez) [inline app/services/albums.js]: what happens if the user wants a specific album? the url would be `api_url/albums/1`

File: app/services/albums.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+const options = {
+  uri: `${api_url}`,
+  headers: {
+    'User-Agent': 'Request-Promise'
+  },
+  json: true
+};
+
+exports.getAlbums = id => {
```
Comment (jersonlopez) [inline app/services/albums.js]: the method isn't only for get albums, also allow get album's photos

File: app/services/albums.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const request_promise = require('request-promise');
+const { logger } = require('express-wolox-logger');
+const api_url = require('../../config').common.albumsApiURL;
```
Comment (jersonlopez) [inline app/services/albums.js]: why this name is in snake_case?

File: app/services/albums.js (hunk 3 above)
Comment (Marcosnr97) [reply app/services/albums.js]: I changed the method to return all albums or a specific album if you send an id, also added a separated method for albums photos

Comment (jersonlopez) [reply app/services/albums.js]: but i think that it could be a single method, the only difference is a uri, what do you think?

**Pull request merged**